cea/plots/sensitivity_demand_graphs.py

```diff
@@ -0,0 +1,106 @@
+# -*- coding: utf-8 -*-
+"""
+===========================
+Graphs for sensitivity_demand.py
+===========================
+
+"""
+from __future__ import division
+
+import pandas as pd
+
+import matplotlib.pyplot as plt
+import numpy as np
+from matplotlib.collections import EllipseCollection
+from matplotlib.backends.backend_pdf import PdfPages
+
+
+__author__ = "Jimeno A. Fonseca"
+__copyright__ = "Copyright 2016, Architecture and Building Systems - ETH Zurich"
+__credits__ = ["Jimeno A. Fonseca"]
+__license__ = "MIT"
+__version__ = "0.1"
+__maintainer__ = "Daren Thomas"
+__email__ = "thomas@arch.ethz.ch"
+__status__ = "Production"
+
+
+def graph(locator, parameters, method, samples):
+    """
+
+    :param locator: locator class
+    :param parameters: list of output parameters to analyse
+    :param method: 'morris' or 'sobol' methods
+    :param samples: number of samples to calculate
+    :return: .pdf file per output_parameter stored in locator.get_sensitivity_plots_file()
+
+    """
+    if method is 'sobol':
+        result_1 = 'ST'
```

Comment:
I wonder if using `result = ('ST', 'conf', 'S1')` and `result = ('mu_star', 'sigma', 'mu_star_conf')` wouldn't be more elegant?